build.gradle

```diff
@@ -167,22 +167,27 @@ eclipseClasspath {
     }
 }
 
+def isTravis = System.getenv("TRAVIS") != null
+def isAppVeyor = System.getenv("APPVEYOR") != null
+def isCi = isTravis || isAppVeyor
+
 task createConfigs {
     description "Sets up the project by obtaining necessary files and configurations points."
     group "Setup"
+    def testTemplateFilename = isCi ? "test.ci.properties" : "test.template.properties"
```

Comment:
I did think about it but decided that even though it'd be more concise, it'd also be less readable. The current code makes it clearer what the two full filenames are.